src/components/calendar/calendar.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import Calendar from './calendar';
+import { CalendarIcon, ExpandIcon } from '../icons';
+
+describe('<Calendar />', () => {
```

Comment:
@andela-akolawole  Nice work so far. However, these tests do not seem comprehensive enough. How about adding tests to check that this component has multiple ui classes, contains span, number of divs it has and  the text it contains.

for example. we can write tests for classes like this
```
expect(wrapper.find('.ui')).to.have.length(3);
expect(wrapper.find('.tab')).to.have.length(1);
expect(wrapper.find('span')).to.have.length(1);
```
Remember the purpose of tests is to be able to look at the test and know what the code is doing.  A better way of communicating what you are trying to achieve to the other developer so it should be comprehensive enough.